server/storage/wal/repair.go

```diff
@@ -67,7 +67,7 @@ func Repair(lg *zap.Logger, dirpath string) bool {
 
 		case errors.Is(err, io.ErrUnexpectedEOF):
 			brokenName := f.Name() + ".broken"
-			bf, bferr := os.Create(brokenName)
+			bf, bferr := fileutil.LockFile(brokenName, os.O_WRONLY|os.O_CREATE, fileutil.PrivateFileMode)
```

Comment:
`fileutil.LockFile` unnecessarily changes the original semantics. It not only creates a new file, but also lock the file. But I agree it isn't a big problem, because the file is closed soon https://github.com/etcd-io/etcd/blob/981061a4958e8d386b025dbecd43af9b974709e2/server/storage/wal/repair.go#L75
```suggestion
			bf, bferr := os.OpenFile(brokenName, os.O_WRONLY|os.O_CREATE, fileutil.PrivateFileMode)
```

Reply:
Updated to use `os.OpenFile`. I agree that it isn't necessary to put a lock on the file in this case of creating a backup WAL file